### ISSUE TITLE
Editorial: consider linking "present" consistently

### ISSUE THREAD

**Issue by annevk** [whatwg/dom#574]
Instead of `present` use `<a for="dictionary member">present</a>`. However, maybe this isn't worth it if dictionaries get recast as Infra maps so we'd have to use maps infrastructure instead...

**Comment by domenic** [comment]
They already are mapped, aren't they? So we can do x["y"] <u>exists</u>.

**Comment by annevk** [comment]
@domenic hmm yeah, maybe we should update https://heycam.github.io/webidl/#idl-dictionaries to encourage the new terminology then?

**Comment by domenic** [comment]
Yeah probably a good idea. Might want to add some stuff to Infra so that any links to "present" get redirected there.
